core/pom.xml

```diff
@@ -194,6 +194,11 @@
             <artifactId>jna</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Our goal is to minimize dependencies, and especially commonly used non-critical dependencies. I wouldn't go so far as to say adding a new dependency as part of removing Guava will upset someone, but @nik9000's tacit point that this here defeats the purpose is correct.